src/hotspot/cpu/x86/x86.ad

```diff
@@ -1693,9 +1694,9 @@ const bool Matcher::match_rule_supported_vector(int opcode, int vlen, BasicType
         return false; // Implementation limitation due to how shuffle is loaded
       } else if (size_in_bits == 256 && UseAVX < 2) {
```

Comment:
Should this be >= 256?

Reply:
The general >= 256 part is taken care of early on in match_rule_supported_vector as below:
  if (!vector_size_supported(bt, vlen)) {
    return false;
  }
The only additional check that is being done here is for float and double  256 bit vectors that are supported on AVX=1 and will pass the vector_size_supported check.
This is because the VectorLoadShuffle cannot be performed for 256 bit vectors on AVX1 platform as it needs "integer" 256 bit instructions which are only available on AVX2.

Reply:
Okay.